src/System.Collections/ref/Configurations.props

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <BuildConfigurations>
       netcoreapp;
+      netstandard;
```

Comment:
This library shouldn't be netstandard any longer. We have a lot of libraries that are still incorrectly configured for netstandard. We no longer build netstandard refs in corefx, for anything that is part of .NET Standard like collections.

@tarekgh the majority of these refs aren't applicable for netfx and so they should essentially get filtered out if they are configured correctly.